RELEASES.md

```diff
@@ -137,17 +137,18 @@ of containerd for every supported version of Kubernetes.
 | 1.23               | 1.6.0+, 1,5.8+     | v1, v1alpha2 |
 | 1.24               | 1.6.4+, 1.5.11+    | v1, v1alpha2 |
 | 1.25               | 1.6.4+, 1.5.11+    | v1, v1alpha2 |
+| 1.26               | 1.6.4+, 1.5.11+    | v1           |
```

Comment:
nak.. v1.26 support is a work in progress...

We have not yet decided what the min release numbers of containerd will be for k8s v1.26.   For the 1.6 branch we've had a couple of breaking issues and I believe it will be 1.6.14 perhaps 15 before we can announce with confidence.

containerd v1.5 only has v1alpha2 apis so it will not work with k8s v1.26

```suggestion
** Note: containerd v.1.5.*, v1.6.*, and v1.7.* will support CRI v1 and v1alpha2 through EOL as those releases will continue to support older versions of k8s, cloud providers, and other clients using CRI v1alpha2. CRI v1alpha2 will be deprecated in v1.7 and removed in containerd v2.
```
```suggestion
| 1.26 | 1.6.tbd, 1.7.tbd | v1, v1alpha2 **  |
```

Reply:
Updated.